e2e/testdata/pod-pvc-template.yaml

```
@@ -3,11 +3,18 @@ apiVersion: v1
 metadata:
   name: {{ .pvcName }}
   namespace: {{ .namespace }}
+{{ if eq .useLabel "true" }}
```

if we change type of `useLabel` to bool, we can write it simply.
```suggestion
{{ if eq .useLabel }}
```

---

Fixed.